src/docfx/SubCommands/DownloadCommand.cs

```diff
@@ -25,6 +25,11 @@ public DownloadCommand(DownloadCommandOptions options)
 
         public void Exec(SubCommandRunningContext context)
         {
+            if (string.IsNullOrWhiteSpace(_options.ArchiveFile))
+            {
+                Logger.LogError("Please provide output file.");
```

Comment:
Change options.ArchiveFile to be a **Required** parameter in the CommandLine settings?

Reply:
There is no **Required** in `ValueOption`